packages/slice-machine/test/src/modules/availableCustomTypes/factory.test.ts

```diff
@@ -6,130 +6,156 @@ describe("[Custom types factory]", () => {
+              "slices": {
+                "config": {
+                  "choices": {},
+                },
+                "fieldset": "Slice Zone",
+                "type": "Slices",
               },
-              "value": [
-                {
-                  "key": "uid",
-                  "value": {
-                    "config": {
-                      "label": "UID",
-                    },
-                    "type": "UID",
-                  },
+              "uid": {
+                "config": {
+                  "label": "UID",
                 },
-              ],
+                "type": "UID",
+              },
             },
-            {
-              "key": "SEO & Metadata",
-              "value": [
-                {
-                  "key": "meta_title",
-                  "value": {
-                    "config": {
-                      "label": "Meta Title",
-                      "placeholder": "A title of the page used for social media and search engines",
-                    },
-                    "type": "Text",
-                  },
+            "SEO & Metadata": {
+              "meta_description": {
+                "config": {
+                  "label": "Meta Description",
+                  "placeholder": "A brief summary of the page",
                 },
-                {
-                  "key": "meta_description",
-                  "value": {
-                    "config": {
-                      "label": "Meta Description",
-                      "placeholder": "A brief summary of the page",
-                    },
-                    "type": "StructuredText",
+                "type": "StructuredText",
+              },
+              "meta_image": {
+                "config": {
+                  "constraint": {
+                    "height": 1260,
+                    "width": 2400,
                   },
+                  "label": "Meta Image",
+                  "thumbnails": [],
                 },
-                {
-                  "key": "meta_image",
-                  "value": {
-                    "config": {
-                      "constraint": {
-                        "height": 1260,
-                        "width": 2400,
-                      },
-                      "label": "Meta Image",
-                      "thumbnails": [],
-                    },
-                    "type": "Image",
-                  },
+                "type": "Image",
+              },
+              "meta_title": {
+                "config": {
+                  "label": "Meta Title",
+                  "placeholder": "A title of the page used for social media and search engines",
                 },
-              ],
+                "type": "Text",
+              },
             },
-          ],
+          },
+          "label": "label",
+          "repeatable": true,
+          "status": true,
         }
       `);
     });
   });
+
+  it("when non repeatable page type", () => {
```

Comment:
❓ What do you prefer for test title? I saw that the 3 tests above use "should ...." and this one "when ..."

I'm use to both but just wanna now, so maybe we can follow the same naming?